client/src/components/CheckBox/CheckBox.tsx

```diff
@@ -9,19 +9,19 @@ import { IconCheckmark } from "src/icons/icons";
 
 import "./CheckBox.scss";
 
-export interface CheckBoxProps {
-  disabled?: boolean; // greys out checkbox and make it non-clickable if true
-  onClick?: (id: string) => void; // (id) => {}
+export type CheckBoxProps = typeof CheckBox.defaultProps & {
```

Comment:
Would be good if you could catch any react components that are exporting their props and clean that up too. Should use the `ComponentProps` utility type where ever the props are needed.